core/src/defaultconfigs/bareos-sd.d/device/FileStorage.conf.in

```diff
@@ -1,4 +1,5 @@
 Device {
+  Description = "Multiplied file device. A connecting Director must have the same Name and MediaType."
```

Comment:
As the name will be the autochanger name 
```suggestion
  Description = "Multiplied file device. A connecting Director must have the same MediaType."
```

Reply:
Should AutoChanger = yes also be set?